Error 'HTTPConnectionPool(host='localhost', port=48678): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f68170d1470>: Failed to establish a new connection: [Errno 111] Connection refused',))' happened on line 215    Mon May 20 17:26:26 2019
Error 'Message: no such window: window was already closed
  (Session info: chrome=74.0.3729.131)
  (Driver info: chromedriver=74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}),platform=Linux 4.4.0-142-generic x86_64)
' happened on line 245    Mon May 20 19:27:28 2019
